planning_scene/src/planning_scene.cpp

```diff
@@ -1185,14 +1187,16 @@ void planning_scene::PlanningScene::setPlanningSceneDiffMsg(const moveit_msgs::P
 
   // process collision object updates
   for (std::size_t i = 0 ; i < scene_msg.world.collision_objects.size() ; ++i)
-    processCollisionObjectMsg(scene_msg.world.collision_objects[i]);
+    ret &= processCollisionObjectMsg(scene_msg.world.collision_objects[i]);
```

Comment:
I'm wondering if this is our of style for MoveIt!'s code base - we have almost no ``&=`` in the code base (except in the rviz plugin, twice) and I found it hard to understand at first. Wouldn't ``+=`` accomplish the same thing but be easier to quickly understand? Its probably just me being ignorant but I'm not used to seeing it